spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -584,26 +584,15 @@ implementations.
 
 ==== Rabbit MQ Binder properties
 
-The binder supports the all Spring Boot properties for Rabbit MQ configuration.
+By default, the binder uses the Spring Boot `ConnectionFactory` and therefore it supports all the Spring Boot configuration options for Rabbit MQ.
+For reference, consult the [Spring Boot documentation](http://docs.spring.io/spring-boot/docs/1.3.3.RELEASE/reference/htmlsingle/#common-application-properties). Rabbit MQ configuration options use the `spring.rabbitmq` prefix.
```

Comment:
Agreed, but on the long run I think we need to match built documentation with release versions of dependencies. Documentation for older builds may end up referencing more recent versions of Spring Boot (e.g. this may end up referencing Spring Boot 1.4 when 1.4 is `current`, which may contain in turn properties that aren't really supported here - it's a minimal risk, so for now I think it is fine).